src/test/scala/uk/gov/hmrc/perftests/itsass/AuthLoginRequests.scala

```diff
@@ -93,7 +93,7 @@ object AuthLoginRequests extends ServicesConfiguration {
     .post(authLoginUrl + "/auth-login-stub/gg-sign-in": String)
     .formParam("authorityId", "")
     .formParam("gatewayToken", "")
-    .formParam("redirectionUrl", s"$serviceUrl/2022/start")
+    .formParam("redirectionUrl", s"$serviceUrl/2022/start/additional-parameters?ClientNino=AA123457A&ClientMTDID=1234567890")
```

Comment:
this url is missing the test only tag technically this means the auth request is dud based on the redirect, but because we're forcing redirects afterwards it is able to recover. Would be best to check what the Journey is doing here.

Reply:
I have added the test-only to the url

Reply:
should be s"$serviceUrl/test-only/2022/additional-parameters?ClientNino=AA123456A&ClientMTDID=1234567890"